src/io/io.jl

```diff
@@ -127,10 +127,18 @@ function save_solution_file(dg::AbstractDg, mesh::TreeMesh, time, dt, timestep,
 
     # Convert to primitive variables if requested
     solution_variables = parameter("solution_variables", "primitive",
-                                   valid=["conservative", "primitive"])
+                                   valid=["conservative", "primitive", "pot"])
     if solution_variables == "conservative"
       data = dg.elements.u
       varnames = varnames_cons(equation)
+    elseif solution_variables == "pot"
```

Comment:
I think `pot` is not descriptive enough. The indicator could be named `pottemp` though, and the related functions to `cons2pottemp` and `varnames_pottemp`